### PR TITLE
feat(kanban): Kanban visibility: Collapsed status

### PR DESCRIPTION
**Pull request by rsanchezbalo**
- [ ] Open kanban and un-collapse all open status, even archived ones. Close the window and re-enter. Verify that the archive is collapsed again.
- [ ] Collapse 1 column and repite to check that it continues collapsed.
- [ ] Change the zoom and check again.
- [ ] Check that the number of user stories indicated is correct.

### CODE REVIEW

File: app/partials/includes/modules/kanban-table.jade [hunk 1]
```diff
@@ -47,14 +37,32 @@ div.kanban-table(
...
                         tg-svg.bulk-action(svg-icon="icon-bulk")
+                    a.option(
+                        href=""
+                        ng-click='foldStatus(s)'
+                        title="{{'KANBAN.TITLE_ACTION_FOLD' | translate}}"
+                        ng-class='{hidden:folds[s.id]}'
+                    )
+                        tg-svg(svg-icon="icon-fold-column")
 
-                    a.option.e2e-archived(
```
Comment (Xaviju) [inline app/partials/includes/modules/kanban-table.jade:51]: If we remove this class we might be broking e2e tests

File: app/styles/modules/kanban/kanban-table.scss [hunk 2]
```diff
@@ -256,3 +268,59 @@ $fold-animation-duration: .1s;
...
+
+    .name {
+        font-weight: bold;
+    }
+
+    .text-holder {
+        color: $color-black900;
+        display: flex;
+        flex-direction: row-reverse;
+        font-size: 14px;
```
Comment (Xaviju) [inline app/styles/modules/kanban/kanban-table.scss:300]: Use the fonts mixin

File: app/styles/modules/kanban/kanban-table.scss [hunk 3]
```diff
@@ -256,3 +268,59 @@ $fold-animation-duration: .1s;
...
+        position: absolute;
+        text-transform: uppercase;
+        top: 0;
+        transform: rotate(-180deg);
+        width: 100%;
+        writing-mode: tb-rl;
+    }
+
+    .name {
+        font-weight: bold;
```
Comment (Xaviju) [inline app/styles/modules/kanban/kanban-table.scss:293]: Use fonts mixins

File: app/styles/modules/kanban/kanban-table.scss [hunk 4]
```diff
@@ -256,3 +268,59 @@ $fold-animation-duration: .1s;
...
+        color: $color-black900;
+        display: flex;
+        flex-direction: row-reverse;
+        font-size: 14px;
+        padding-top: .5rem;
+    }
+
+    .ammount {
+        color: $color-link-tertiary;
+        font-size: 12px;
```
Comment (Xaviju) [inline app/styles/modules/kanban/kanban-table.scss:306]: Use the fonts mixin

File: app/styles/modules/kanban/kanban-table.scss [hunk 5]
```diff
@@ -256,3 +268,59 @@ $fold-animation-duration: .1s;
...
+        padding-top: .5rem;
+    }
+
+    .archived {
+        color: $color-link-tertiary;
+        font-weight: bold;
```
Comment (Xaviju) [inline app/styles/modules/kanban/kanban-table.scss:312]: Use the fonts mixin

**Review by Xaviju — CHANGES_REQUESTED** [review]

File: app/partials/includes/modules/kanban-table.jade (hunk 1 above)
Comment (rsanchezbalo) [reply app/partials/includes/modules/kanban-table.jade:51]: This button does not exist anymore, should i look for it on the e2e and remove the test?

Comment (rsanchezbalo) [reply app/partials/includes/modules/kanban-table.jade:51]: removed

**Review by Xaviju — APPROVED** [review]

**Pull request merged**